<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageChecker</class>
 <widget class="QMainWindow" name="PageChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="widget_list_of_sites">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>231</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;





</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_site">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_remove_site">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_update">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_font_plus">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_font_minus">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_color_gray">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_color_white">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeWidget" name="widget_items_of_site">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>871</width>
      <height>640</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>url</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>content</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
